component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPConstants.java

```diff
@@ -86,6 +86,7 @@
     public static final String ERROR_PAGE = "smsotpauthenticationendpoint/smsotpError.jsp";
     public static final String MOBILE_NUMBER_REQ_PAGE = "MobileNumberRegPage";
     public static final String MOBILE_NUMBER = "MOBILE_NUMBER";
+    public static final String REQUEST_USER_MOBILE = "requestUserMobile";
```

Comment:
IMO this variable name implies a boolean value. shall we rename it to requestedMobileNumber or something similar?

Reply:
This is also updated accordingly